migrations/20200411213654_create-weather.js

```diff
@@ -1,8 +1,18 @@
-
 exports.up = function(knex) {
-  
+  return knex.schema.createTable('weather', (table) => {
+    table.string('zipcode', 5).notNullable().primary(); /* PK */
+    table.decimal('day_max_temp', 5, 2); /* Length, Precision might be off */
```

Comment:
The api returns temperature in Kelvin with two decimal points of precision. Do we want to store it as kelvin and allow user to select C/F or do we want to convert it on input?

Reply:
We should definitely have it convert on input. I don’t see the benefit of storing two different conversion values.

Reply:
We wouldn't store two values, we'd store the Kelvin, and then based an a user toggle they would see F/C, but since our weather api is usa zipcode based, to me it makes sense just to store it as Fahrenheit.

Reply:
Wait actually it depends on what we might use more. If are always going to display probably in Farenheight, then we should store as F? Unless we add a user settings (table? Column? Idk) for metric. In which case storing it as kelvin should be fine to make comparisons etc. when we pull the data and we should convert.

Reply:
Actyally you are right. Storing it as Farenheight would probably be our best bet, since we won’t need that many stored temperature to api temperature comparisons

Reply:
Is the Kelvin payload from the weather API in decimal, too? Not sure if this is useful info, but in the plant table, the temperatures pulled from the Trefle API are for both C/F. If we want to make comparisons, in either case we will probably be making conversions from Kelvin to something else. Should I modify anything to the schema if we chose to store the value as Fahrenheit?

Reply:
Oh, we can change the units on the API call itself by adding
`&units=imperial` or `&units=metric`
I don't think we need to change anything in the schema, decimal(5,2) will give us -999.99 to 999.99 which will fit the realistic temperature of all three units.